bee-node/src/node/error.rs

```diff
@@ -13,6 +13,9 @@ pub enum Error {
     #[error("Storage backend operation failed: {0}.")]
     StorageBackend(Box<dyn std::error::Error>),
 
-    #[error("Network initialization failed. Cause: {}", .0)]
+    #[error("Network initialization failed. Cause: {0}")]
```

Comment:
I seem to remember this should be lowercase :P

Reply:
You mean the error messages should all be lowercase without punctuation? Is it that what you remember? Just making sure... :smile:

Reply:
I remember something like:
- First character should not be capitalized
- No dot at the end
but the details are blurry :trollface:

Reply:
haha, for me as well, but I think I remember the same, and changed accordingly :+1: (I hope)